strictequal.js

```diff
@@ -0,0 +1,13 @@
+const strictEqual = (value1, value2) => {
+  let value1type = typeof value1
+  let value2type = typeof value2
+  if (value1type == value2type && value1 == value2) {
```

Comment:
There's no need to create extra variable here you could just do the `typeof` right in the if condition

```JavaScript
if (typeof value1 == typeof value2 && value1 == value2) {
```

Reply:
I was trying to do the type of in the if but couldn't seem to get the syntax right.  eslint kept barking at me.